### DISCUSSION TITLE
Scrobble point

### DISCUSSION THREAD

**Discussion by codeinabox** [navidrome/navidrome#2315] [General]
Does Navidrome have support for a scrobble point like Simple Scrobbler? When I was listening to my collection using Substreamer, I noticed that Navidrome scrobbled tracks that I immediately skipped, whilst I'd rather only count tracks where I listened to 75%

**Comment by dweymouth** [comment]
The decision of when to scrobble is down to the individual Subsonic clients. Maybe bring this up with the Substreamer dev that tracks should only scrobble if they've been listened to for some threshold of time (Last.fm recommends 50+% OR 4+ minutes of play).
